src/dscanner/analysis/base.d

```diff
@@ -81,7 +81,7 @@ protected:
 		override void visit(const T structDec)
 		{
 			inAggregate = true;
-			structDec.accept(this);
+			() @trusted { structDec.accept(this); } ();
```

Comment:
I don't think this accept call can generally be `@trust`ed, this should probably stay `@system`